framework/tasks/handlers.py

```diff
@@ -43,7 +43,7 @@ def enqueue_task(signature):
     try:
         if signature not in queue:
             queue.append(signature)
-    except RuntimeError:
+    except (RuntimeError, AttributeError):
```

Comment:
Can you explain why an `AttributeError` needs to be caught here? It looks like `queue` should always be a list at this point.